source/2020-08-28-the-ember-times-issue-162.md

```diff
@@ -15,7 +15,7 @@ READMORE
 
 ---
 
-## [A new and easy way to add syntax highlighting to your code blocks, with `ember-showdown-prims` 🔦](https://twitter.com/real_ate/status/1297892167181099010)
+## [A new way to add syntax highlighting to your code blocks, with `ember-showdown-prims` 🔦](https://twitter.com/real_ate/status/1297892167181099010)
```

Comment:
Hey, welcome back! 😄   One small thing. Also, don't forget to add your intro blurb and your name to the top and bottom.

```suggestion
## [A new way to add syntax highlighting to your code blocks, with `ember-showdown-prism` 🔦](https://twitter.com/real_ate/status/1297892167181099010)
```